compiler/damlc/tests/daml-test-files/IncompleteAdditiveInstance.daml

```diff
@@ -0,0 +1,17 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- @ERROR range=14:10-14:23; • No explicit implementation for; either ‘-’ or ‘negate’; • In the instance declaration for ‘Additive Expr’
```

Comment:
the unicode might cause issues if it’s rendered differently on Windows (not sure if it is, maybe it’s fine). in that case I’d just start shorten it to `No explicit implementation for; either ‘-’ or ‘negate’`. Together with the range that still tests the right thing.

Reply:
ah good point, I fixed it now. Thanks!

Reply:
ah good point, but it seems to have gone through :)

Reply:
if it goes through even better :)